lib/PuppeteerSharp/Page.cs

```diff
@@ -169,6 +170,10 @@ private Page(Session client, Target target, FrameTree frameTree, bool ignoreHTTP
                 {
                     cookie.Url = Url;
                 }
+                if (cookie.Url == "about:blank")
+                {
+                    throw new Exception($"Blank page can not have cookie \"{cookie.Name}\"");
```

Comment:
I was thinking about the same thing in my previous PR. Definitely we shouldn't use an Exception, maybe we can create a PuppeterException for general exceptions.

On the other hand it could sound as an InvalidArgumentException.

Reply:
Don't forget about this guy @Meir017 :)